CustomGauge/src/main/java/pl/pawelkleczkowski/customgauge/CustomGauge.java

```diff
@@ -153,6 +173,7 @@ protected void onDraw(Canvas canvas) {
             }
         }
 
+        if(mLightEffectActive) invalidate();
```

Comment:
please reformat code (missing space here); use braces for if conditions, even if its only one instruction, move it to new line